sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_common/_base_prompty_eval.py

```diff
@@ -287,8 +287,7 @@ def _extract_needed_tool_definitions(
                     elif tool_name:
                         # This is a regular function tool from converter
                         tool_definition_exists = any(
-                            tool.get("name") == tool_name and tool.get("type", "function") == "function"
-                            for tool in tool_definitions_expanded
+                            tool.get("name") == tool_name for tool in tool_definitions_expanded
```

Comment:
Do we need to check whether this tool is in `needed_tool_definitions` or not? We can also do it at downstream